sched/signal/sig_default.c

```diff
@@ -225,6 +227,30 @@ static void nxsig_abnormal_termination(int signo)
   group_kill_children(rtcb);
 #endif
 
+#ifndef CONFIG_DISABLE_PTHREAD
+  if ((rtcb->flags & TCB_FLAG_TTYPE_MASK) == TCB_FLAG_TTYPE_PTHREAD &&
```

Comment:
How can this recursive call happen, can you explain ?

Reply:
hi, pussuw, this depends on another modify，i'll remove to other PR

Reply:
https://github.com/apache/nuttx/pull/13791